spec/models/test_track/remote/split_registry_spec.rb

```diff
@@ -22,7 +22,7 @@
 
     begin
       Rails.cache.clear
-    rescue Errno::ENOENT # rubocop:disable Lint/HandleExceptions
+    rescue Errno::ENOENT
       # This is fine
```

Comment:
You're now allowed to comment about _why_ you aren't handling an exception, and it doesn't violate the cop.